scripts/icu/63.1-min-static-data/script.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+# Build ICU common package (libicuuc.a) with data file separate and with support for legacy conversion and break iteration turned off in order to minimize size
+
+MASON_NAME=icu
+MASON_VERSION=63.1-min-static-data
+MASON_LIB_FILE=lib/libicuuc.a
```

Comment:
@springmeyer can one put more than one library on this line?

Reply:
This is used to check that the install worked. It only supports a single file and can be anything that exists after the build completes. So, the way you have it here is 👍. No need (or ability) to point to multiple files and not pointing to the other libraries  is okay - they will still get installed.